### ISSUE TITLE
Doc suggests using releases, but repo has no releases

### ISSUE THREAD

**Issue by rofrankel** [bufbuild/protovalidate-cc#49]
There are no releases at https://github.com/bufbuild/protovalidate-cc/releases, which makes the instructions [here](https://github.com/bufbuild/protovalidate-cc/blob/main/README.md?plain=1#L48) and [here](https://github.com/bufbuild/protovalidate-cc/blob/main/README.md?plain=1#L64) from the README hard to follow.

Is this because the code isn't yet in a release-ready state?

**Comment by Alfus** [comment]
Added a release
